packages/pigeon/lib/java_generator.dart

```diff
@@ -178,14 +171,13 @@ String _javaTypeForDartType(String datatype) {
   return _javaTypeForDartTypeMap[datatype];
 }
 
-String _mapGetter(Field field, List<Class> classes, String mapName) {
+String _mapCaster(Field field, List<Class> classes, String varName) {
```

Comment:
uber nit: variableName

Reply:
Na you are right, doesn't even make sense.  Fixed it.